pkg/config/config.go

```diff
@@ -325,6 +326,28 @@ type Connect struct {
 	Pid int `def:"0" desc:"PID of the process you want to profile. Pass -1 to profile the whole system (only supported by ebpfspy)" mapstructure:"pid"`
 }
 
+type EBPFSpy struct {
+	LogLevel  string `def:"info" desc:"log level: debug|info|warn|error" mapstructure:"log-level"`
+	NoLogging bool   `def:"false" desc:"disables logging from pyroscope" mapstructure:"no-logging"`
+
+	// Spy configuration
+	ApplicationName string `def:"" desc:"application name used when uploading profiling data" mapstructure:"application-name"`
+	SampleRate      uint   `def:"100" desc:"sample rate for the profiler in Hz. 100 means reading 100 times per second" mapstructure:"sample-rate"`
+
+	// Remote upstream configuration
+	ServerAddress          string        `def:"http://localhost:4040" desc:"address of the pyroscope server" mapstructure:"server-address"`
+	AuthToken              string        `def:"" desc:"authorization token used to upload profiling data" mapstructure:"auth-token"`
+	UpstreamThreads        int           `def:"4" desc:"number of upload threads" mapstructure:"upstream-threads"`
+	UpstreamRequestTimeout time.Duration `def:"10s" desc:"profile upload timeout" mapstructure:"upstream-request-timeout"`
+
+	Tags map[string]string `name:"tag" def:"" desc:"tag in key=value form. The flag may be specified multiple times" mapstructure:"tags"`
+
+	Pid                int    `def:"-1" desc:"PID of the process you want to profile. Pass -1 to profile the whole system" mapstructure:"pid"`
+	DetectSubprocesses bool   `def:"false" desc:"makes pyroscope keep track of and profile subprocesses of the main process" mapstructure:"detect-subprocesses"`
+	SymbolCacheSize    int    `def:"256" desc:"todo" mapstructure:"symbol-cache-size"`
```

Comment:
```suggestion
	SymbolCacheSize    int    `def:"256" desc:"symbol cache in megabytes" mapstructure:"symbol-cache-size"`
```

Reply:
It is not megabytes it is number of processes to keep symbols parsed in memory

Reply:
```suggestion
	SymbolCacheSize    int    `def:"256" desc:"max size of symbols cache (1 entry per process)" mapstructure:"symbol-cache-size"`
```